packages/fhir-converter/src/templates/cda/Sections/Medication.hbs

```diff
@@ -10,42 +10,44 @@
-                                        {{>References/MedicationRequest/requester.hbs ID=(generateUUID (toJsonString medReq.supply)) REF=(concat 'Organization/' orgId.Id)}}
+                                {{#if entryRelationship.supply.author.assignedAuthor.representedOrganization}}
+                                    {{#with (evaluate 'Utils/GenerateOrganizationId.hbs' obj=entryRelationship.supply.author.assignedAuthor.representedOrganization) as |orgId|}}
+                                        {{>Resources/Organization.hbs org=entryRelationship.supply.author.assignedAuthor.representedOrganization ID=orgId.Id}},
+                                        {{>References/MedicationRequest/requester.hbs ID=(generateUUID (toJsonString entryRelationship.supply)) REF=(concat 'Organization/' orgId.Id)}}
                                     {{/with}}
                                 {{/if}}
-                            {{else if (and medReq.supply medEntry.substanceAdministration.author.assignedAuthor)}}
+                            {{else if (and entryRelationship.supply medEntry.substanceAdministration.author.assignedAuthor)}}
                                 {{#with (evaluate 'Utils/GeneratePractitionerId.hbs' obj=medEntry.substanceAdministration.author.assignedAuthor) as |practitionerId|}}
                                     {{>Resources/Practitioner.hbs practitioner=medEntry.substanceAdministration.author.assignedAuthor ID=practitionerId.Id}},
-                                    {{>References/MedicationRequest/requester.hbs ID=(generateUUID (toJsonString medReq.supply)) REF=(concat 'Practitioner/' practitionerId.Id)}}
+                                    {{>References/MedicationRequest/requester.hbs ID=(generateUUID (toJsonString entryRelationship.supply)) REF=(concat 'Practitioner/' practitionerId.Id)}}
                                 {{/with}}
                                 {{#if medEntry.substanceAdministration.author.assignedAuthor.representedOrganization}}
                                     {{#with (evaluate 'Utils/GenerateOrganizationId.hbs' obj=medEntry.substanceAdministration.author.assignedAuthor.representedOrganization) as |orgId|}}
                                         {{>Resources/Organization.hbs org=medEntry.substanceAdministration.author.assignedAuthor.representedOrganization ID=orgId.Id}},
-                                        {{>References/MedicationRequest/requester.hbs ID=(generateUUID (toJsonString medReq.supply)) REF=(concat 'Organization/' orgId.Id)}}
+                                        {{>References/MedicationRequest/requester.hbs ID=(generateUUID (toJsonString entryRelationship.supply)) REF=(concat 'Organization/' orgId.Id)}}
                                     {{/with}}
                                 {{/if}}
+                            {{else if (contains (toJsonString entryRelationship.observation.templateId) '2.16.840.1.113883.10.20.22.4.19')}}
```

Comment:
I can't tell what the OID is without looking up online. This is a yellow flag, as it increases dependency on the internet and external services. Maybe we can start moving those shared constants on a single file?